README.md

```diff
@@ -168,8 +168,10 @@ The packed variables will be sampled all together in the Markov-chain based solv
 Moreover, packed variables usually indicate nontrivial correlations between their distributions. In the future, it will be interesting to learn such correlation so that one can sample the packed variables more efficiently.
 
 # Parallelization
+MCIntegration supports both MPI and multi-thread parallelization. You can even mix them if necessary.
 
-MCIntegration supports MPI parallelization. To run your code in MPI mode, simply use the command
+## MPI
+To run your code in MPI mode, simply use the command,
 ```bash
 mpiexec julia -n #NCPU ./your_script.jl
```

Comment:
Typo; the command should be: `mpiexec -n #NCPU julia ./your_script.jl`.

Perhaps also mention the alternative Julia wrapper `mpiexecjl` / link to the [docs](https://juliaparallel.org/MPI.jl/stable/usage/#Julia-wrapper-for-mpiexec)?

Reply:
Great, good to know!